### PR TITLE
Update magianobjectives.lua

### PR DESCRIPTION
**Pull request by Daywhite83**
Add missing trials for Empyrean Armors.

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?
Adds Magian trials for Empyrean armors. Will post screenshots of this working in the other thread.

## Steps to test these changes
Visit the Blue Magian moogle.
Accept a trial using an empyrean base (NQ or +1) by trading it to the moogle.
trade corresponding seal or stone/jewel/coin etc. to the delivery box next to the moogle.
trade armor back to the moogle
get the reward.

### CODE REVIEW

**Review by Daywhite83 — COMMENTED** [review]
Took out the line the sanity check did not like,

**Comment by Daywhite83** [comment]
I Don't understand why it failed on 
build / MultiInstance_Startup_Checks_Linux (pull_request)
(On Line 27 it said) Error: 22 03:24:52:159][login][error] make_listen_bind: bind failed (socket #6, port 54230, code 98)! (makeListenBind_tcp:830)
(On Line 94 it said) Error: Process completed with exit code 255.

**Comment by Daywhite83** [comment]
All other checks passed

File: scripts/globals/magianobjectives.lua [hunk 1]
```diff
@@ -683,6 +683,243 @@ xi.magian.trials =
...
+[4248] = { check = checks.checkTradeTrials, reqs = { itemId = set{ 3222 } } }, -- +1 -> +2 (Stone of Balance)
+[4268] = { check = checks.checkTradeTrials, reqs = { itemId = set{ 3229 } } }, -- +1 -> +2 (Card of Voyage)
+
+-- Dragoon
+[4169] = { check = checks.checkTradeTrials, reqs = { itemId = set{ 3123 } } }, -- 0 -> +1 (Lancer's Seal: Hd.)
+[4349] = { check = checks.checkTradeTrials, reqs = { itemId = set{ 3143 } } }, -- 0 -> +1 (Lancer's Seal: Bd.)
+[4329] = { check = checks.checkTradeTrials, reqs = { itemId = set{ 3163 } } }, -- 0 -> +1 (Lancer's Seal: Hn.)
+[4189] = { check = checks.checkTradeTrials, reqs = { itemId = set{ 3183 } } }, -- 0 -> +1 (Lancer's Seal: Lg.)
+[4209] = { check = checks.checkTradeTrials, reqs = { itemId = set{ 3203 } } }, -- 0 -> +1 (Lancer's Seal: Ft.)
+[4229] = { check = checks.checkTradeTrials, reqs = { itemId = set{ 3211 } } }, -- +1 -> +2 (Coin of Vision)
```
Comment (WinterSolstice8) [inline scripts/globals/magianobjectives.lua]: comment should be Card of Vision

**Review by Daywhite83 — COMMENTED** [review]
Fixed the error with Dragoon Head 3211 -> 3213 (Card of Vision), Thanks for catching that.

**Review by WinterSolstice8 — APPROVED** [review]
checks out to the best of my knowledge

**Pull request merged**